vmdb/app/assets/javascripts/miq_application.js

```diff
@@ -50,14 +50,10 @@ function miqOnLoad() {
   if (typeof miqEditor != "undefined") {
     miqEditor.refresh();
   }
-  // Focus on certain tree node, if set
-  if (typeof miq_tree_focus == "string") {
-    eval(miq_tree_focus);
-  }
```

Comment:
I didn't found any other place, where `miq_tree_focus` would be used, so I guess it's not used anymore.